pkg/v1/cli/command/core/update.go

```diff
@@ -31,6 +32,12 @@ var updateCmd = &cobra.Command{
 		"group": string(cliv1alpha1.SystemCmdGroup),
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if config.IsFeatureActivated(config.FeatureContextAwareCLIForPlugins) {
+			// TODO: Implement inline update of tanzu CLI when `context-aware-cli-for-plugins` feature flag is enabled
+			log.Warning("Updating CLI and plugins is not supported when `context-aware-cli-for-plugins` feature flag is enabled")
```

Comment:
The log message may not make sense to users especially now that the feature is activated by default.
Maybe something like:

log.Warning("CLI self-update is currently not support. For updating plugins please use the `plugin sync` command")